pl_engine/pl_server/src/main/java/com/cubrid/plcsql/compiler/visitor/JavaCodeWriter.java

```diff
@@ -827,6 +833,127 @@ public CodeToResolve visitExprBuiltinFuncCall(ExprBuiltinFuncCall node) {
         return applyCoercion(node.coercion, tmpl, node.ctx);
     }
 
+    private static String getNormalCallStr(String funcName, int argsLen) {
+
+        String hostVarStr;
+        if (SymbolStack.noParenBuiltInFunc.indexOf(funcName) >= 0) {
+            assert argsLen == 0;
+            hostVarStr = "";
+        } else {
+            hostVarStr = getHostVarsStr(argsLen);
+        }
+
+        return funcName + hostVarStr;
+    }
+
+    private static String getHostVarsStr(int len) {
+        if (len == 0) {
+            return "()";
+        } else {
+            String[] arr = new String[len];
+            Arrays.fill(arr, "?");
+            return "(" + String.join(", ", arr) + ")";
```

Comment:
```suggestion
return String.format("(%s)", String.join(", ", array))
```
아래에서 String.format 메서드를 주로 사용함으로 다음과 같이 수정해도 좋을 거 같습니다.

Reply:
네. 제안하신 모양이 좀 더 읽기 좋네요. 
수정하겠습니다.